moviego.go

```diff
@@ -88,6 +90,13 @@ func (V Video) ResizeByWidth(requiredWidth int64) Video {
 	return V
 }
 
+func (V Video) Hls() Video {
+	V.addKwArgs("f", "hls")
+	V.addKwArgs("hls_time", "3")
```

Comment:
Default for this is 2 in ffmpeg, is there a specific reason to set this to 3?

Reply:
i don't think so... maybe i could add a parameter to Hls method and overwrite the default only if necessary!

Reply:
For me that would be clearer to have the same behavior as when using ffmpeg directly (see https://ffmpeg.org/ffmpeg-formats.html#Options-26).